# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-align/blocks/align-content-block-011.html.ini
# Commit: b0b878dae244
# Full Hash: b0b878dae244416c3b4799f43ee2f5176f3d3fc8
# Author: fantasai <fantasai.cvs@inkedblade.net>
# Date: 2024-02-28 04:30:28
# Regressor Bug: 1765615
# File Overlap Count: 1
# Description:
#   Bug 1684236 - Implement 'align-content' on block containers. r=layout-reviewers,jfkthame
#   
#   This is Fantasai's original patch, massively simplified:
#   
#    * We now can switch whether we're a BFC dynamically (bug 1765615), which
# ==============================================================================

diff -r 6b8582441a76 -r b0b878dae244 testing/web-platform/meta/css/css-align/blocks/align-content-block-011.html.ini
--- a/testing/web-platform/meta/css/css-align/blocks/align-content-block-011.html.ini	Tue Feb 27 11:35:54 2024 -0500
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,48 +0,0 @@
-[align-content-block-011.html]
-  [.test 10: unsafe end]
-    expected: FAIL
-
-  [.test 11: safe start]
-    expected: FAIL
-
-  [.test 6: flex-start]
-    expected: FAIL
-
-  [.test 15: space-between]
-    expected: FAIL
-
-  [.test 7: flex-end]
-    expected: FAIL
-
-  [.test 9: unsafe center]
-    expected: FAIL
-
-  [.test 1: start]
-    expected: FAIL
-
-  [.test 12: safe center]
-    expected: FAIL
-
-  [.test 8: unsafe start]
-    expected: FAIL
-
-  [.test 16: space-around]
-    expected: FAIL
-
-  [.test 2: center]
-    expected: FAIL
-
-  [.test 4: baseline]
-    expected: FAIL
-
-  [.test 3: end]
-    expected: FAIL
-
-  [.test 13: safe end]
-    expected: FAIL
-
-  [.test 14: space-evenly]
-    expected: FAIL
-
-  [.test 5: last baseline]
-    expected: FAIL